docs/kratos/session-management/10_session-lifespan.mdx

```diff
@@ -80,8 +136,12 @@ session:
 </Tabs>
 ````
 
+:::note
+
 If `max-age` is set as a part of the `Set-Cookie` header, the browser deletes the cookie when it reaches the age defined in
 `max-age`.
 
 When `max-age` is not set, the browser deletes the cookie when the session ends. The session ends when the set session lifespan
 expires, or when the browser is shut down by the user.
```

Comment:
Is this applicable here, since we always have a max-age right?

Reply:
Apparently not, when the persistent cookie configuration is turned off, it isn't - 

```go
	if s.r.Config().SessionPersistentCookie(ctx) {
		if session.ExpiresAt.IsZero() {
			cookie.Options.MaxAge = int(s.r.Config().SessionLifespan(ctx).Seconds())
		} else {
			cookie.Options.MaxAge = int(time.Until(session.ExpiresAt).Seconds())
		}
	}
```

Reply:
Interesting, so inside of the project configuration this is configurable to Ory Network users?